### ISSUE TITLE
Ignore hidden folders in node_modules

### ISSUE THREAD

**Issue by kevgo** [charlierudolph/dependency-lint#30]
Alternative package managers like [ied](https://github.com/alexanderGugel/ied) or [pnpm](https://github.com/rstacruz/pnpm) create hidden folders in `node_modules`, comparable to Git's file-based object storage in `.git`. These folders should be ignored by dependency-lint, since they are referenced via symlinks as needed (see issue #29).

In real life, this causes a false positive error:

```
devDependencies:
    ✖ .cas (missing)
        used in files:
          bin/spec
          bin/tests
```

Dependency-lint finds binaries in `node_modules/.cas`, which it also finds being used, and assumes `.cas` is an unused NPM module. The solution is to ignore folders in `node_modules` whose name begins with a period.
